src/executors/linux.yml

```diff
@@ -5,10 +5,7 @@
 
 # How to author Executors: https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-executors
 description: >
-  Default node executor
-docker:
-  - image: node:<<parameters.tag>>
-parameters:
-  tag:
-    default: "12"
-    type: string
+  Linux os
+
+machine: # executor type
+  image: ubuntu-2004:202010-01 # # recommended linux image - includes Ubuntu 20.04, docker 19.03.13, docker-compose 1.27.4
```

Comment:
I was actually wondering about this in the previous PR. Where is this recommended? Should we use https://circleci.com/docs/2.0/circleci-images/#circleci-base-image ?

Reply:
@amphro I like using this image because it comes preloaded with `nvm` which is super helpful. Also, it's the recommended image for linux: https://circleci.com/docs/2.0/configuration-reference/#available-machine-images